pages/common/gmssl.md

```diff
@@ -0,0 +1,36 @@
+# gmssl
+
+> GmSSL is a crypto toolkit supporting SM1, SM2, SM3, SM4, SM9, and ZUC/ZUC256.
+> More information: <http://gmssl.org/>.
+
+- Generate an SM3 hash for a file:
+
+`gmssl sm3 {{path/to/file}}`
+
+- Encrypt a file using the SM4 cipher:
+
+`gmssl sms4 -e -in {{path/to/file}} -out {{path/to/file}}.sms4`
```

Comment:
```suggestion
`gmssl sms4 -e -in {{path/to/file}} -out {{path/to/file.sms4}}`
```

Refer to [style-guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#token-syntax).